go.mod

```diff
@@ -1,35 +1,37 @@
 module github.com/grafana/xk6-output-influxdb
 
-go 1.18
+go 1.20
```

Comment:
Drop back to 1.19. Some tooling does not support 1.20 yet, and we're keeping in step with k6 on this one.

Reply:
Done